src/components/TransactionsList/TransactionsList-redux.tsx

```diff
@@ -2,12 +2,11 @@ import React from 'react';
 import { useMappedState } from 'redux-react-hook';
 
 import TransactionListContainer from './TransactionsList-container';
-import { getBlockchainByDatasource } from '../../store/reducers/dataSource';
 import { Block } from "../../types";
 
 const TransactionListRedux = () => {
     const mapState = (state: any): { blokchain: Block[] } => ({
```

Comment:
dodaj typ :) (State)

Reply:
This will be typed in next task when I know what type of data will be stored.